alerting/ingest-alerting/integrations-ingest-alerting.tf

```
@@ -0,0 +1,16 @@
+resource "signalfx_webhook_integration" "sfx_integration" {
+  name    = format("%s", var.suffix)
```

I notice this is already the case on previous "proxy" version but I would prefer to:
- hardcode a common name like `ingest alerting` used as __prefix__
- provide a __suffix__ variable empty string by default to customize it as we did for cloud based integrations

Moreover in the current state using `format` function is useless:

```suggestion
  name    = var.suffix
```